driver-mapping/src/main/java/com/datastax/driver/mapping/PassThroughNamingStrategy.java

```diff
@@ -0,0 +1,25 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.mapping;
+
+public class PassThroughNamingStrategy implements NamingStrategy {
```

Comment:
This is not used anywhere, do we need to provide it? Looks simple enough to reimplement if someone needs it.